Sources/FoundationEssentials/ProcessInfo/ProcessInfo+API.swift

```diff
@@ -0,0 +1,104 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if !FOUNDATION_FRAMEWORK
+
+/// A collection of information about the current process.
+public final class ProcessInfo: Sendable {
```

Comment:
Are there any compelling reasons for this to be a value type (with reference semantics as an implementation detail) instead of a reference type?

Reply:
We’re trying to maintain source compatibility with Darwin where possible, and this is a reference type there.